cmd/ccboc/main.go

```diff
@@ -121,6 +121,58 @@ func getCalculationID(c *cli.Context) error {
 	return nil
 }
 
+func getCalculationResult(c *cli.Context) error {
+	teff := c.Float64("teff")
+	logG := c.Float64("logG")
+
+	fmt.Println(globalConfig.APIURL + "/calculations/results?teff=" + fmt.Sprintf("%0.1f", teff) + "&logg=" + fmt.Sprintf("%0.1f", logG))
+
+	body, responseError, err := request("GET", globalConfig.APIURL+"/calculations/results?teff="+fmt.Sprintf("%0.1f", teff)+"&logg="+fmt.Sprintf("%0.1f", logG), bytes.NewBuffer(nil))
+	if err != nil {
+		logrus.WithError(err).Fatal("error while perfoming the http request")
+	} else if responseError != nil {
+		logrus.WithFields(logrus.Fields{"message": responseError.Message, "status_code": responseError.StatusCode}).Fatal("errors occurred")
+	}
+
+	fileName := fmt.Sprintf("%0.1f___%0.1f", teff, logG)
+
+	file, err := os.Create(fileName)
+
+	defer file.Close()
+
+	_, err = file.WriteString(string(body))
+	if err != nil {
+		logrus.WithError(err).Fatal("Couldn't write the calculation into the file.")
```

Comment:
The error message is wrong. Those data are not a `calculation` but bytes from a tar file.